content/blog/data-science-in-the-cloud/index.md

```diff
@@ -0,0 +1,123 @@
+        "Statement": {
+            "Effect": "Allow",
+            "Principal": "*",
+            "Action": ["s3:GetObject"],
+            # Policy refers to bucket explicitly
+            "Resource": [pulumi.Output.concat("arn:aws:s3:::", site_bucket.id, "/*")]
+        },
+    })
+
+    # Export the website URL
+    pulumi.export("website_url", site_bucket.website_endpoint)
+```
+
+We can write additional functions to [deploy and destroy](https://nbviewer.jupyter.org/github/pulumi/automation-api-examples/blob/main/python/pulumi_via_jupyter/automation_api.ipynb#Automating-your-deployment) the S3 static website. We deploy the stack by calling the `deploy_project` function and pass it the `s3_static_site` function, the Pulumi AWS plugin, and the AWS region.
+
+```python
+s3_site = deploy_project("my_first_project",
+                         s3_static_site,
+                         plugins=[("aws", "v3.23.0")],
+                         config={"aws:region": auto.ConfigValue(value="us-west-2")})
+```
+
+You can view the resources that the function created in two ways. The standard method is to open the AWS console to see your resources.
+
+![AWS Console](aws_console.png)
+
+You can also view them with the [Pulumi console](https://app.pulumi.com) as a resource graph.
+
+![Pulumi resource graph](pulumi_resource_graph.png)
+
+The Pulumi console also displays updates to the static site and the changes made.
+
+![Pulumi updates](pulumi_updates.png)
+
+Now that the static site has been deployed, we want to get the site URL. The `s3_static_site` function exports the website url. Pulumi returns the s3_site as an object with [outputs]({{< relref "/docs/intro/concepts/inputs-outputs" >}}) that we use to retrieve the URL. There are several ways to check if the site is working, but a simple way is to open the URL in a web browser.
+
+```python
+import webbrowser
+
+outputs = s3_site.outputs()
+
+webbrowser.open(url)
+```
+
+![Browser](browser.png)
+
+## Data Science Your Way
+
+In this simple example, we demonstrated how to create a static website from an S3 bucket, write content into the bucket, and set the access policy as a Python function. We called the function, which created a stack for the website. The stack is returned as a Python object with the site URL as an output. You can manage the site and view updates using the Pulumi console.
+
+Because it's code, you can create a separate module containing all the functions to create your infrastructure, import it into your notebooks, and reuse it. Pulumi lets you perform analysis and build models within a familiar environment while reducing the burden of managing infrastructure. You can find the complete example on [Github](https://github.com/pulumi/automation-api-examples/tree/main/python/pulumi_via_jupyter) including a Jupyter notebook for [database migration](https://github.com/pulumi/automation-api-examples/blob/main/python/pulumi_via_jupyter/database_migration.ipynb).
```

Comment:
```suggestion
Because it's code, you can create a separate module containing all the functions to create your infrastructure, import it into your notebooks, and reuse it. Pulumi lets you perform analysis and build models within a familiar environment while reducing the burden of managing infrastructure. You can find the complete example on [GitHub](https://github.com/pulumi/automation-api-examples/tree/main/python/pulumi_via_jupyter) including a Jupyter notebook for [database migration](https://github.com/pulumi/automation-api-examples/blob/main/python/pulumi_via_jupyter/database_migration.ipynb).
```